api/src/stampy_chat/chat.py

```diff
@@ -340,9 +344,38 @@ def run_query(session_id: str, query: str, history: List[Dict], settings: Settin
         prompt=make_prompt(settings, chat_model, callbacks),
         memory=make_memory(settings, history, callbacks)
     )
+
     if followups:
         chain = chain | StampyChain(callbacks=callbacks)
+
     result = chain.invoke({"query": query, 'history': history}, {'callbacks': []})
+
+    #Validate results
+    contexts=[c['text'] for c in get_top_k_blocks(query, 5)]
```

Comment:
I'm guessing this shouldn't be executed for each query. How about adding a flag to the settings object?

Reply:
I think that this is required for each query because `tonic_validate`, checks to see if the LLM's answer is consistent with the context that was retrieved.

Reply:
yes, but should it always do that, is the question. This makes the query slower, so for now I'd just use it for testing, rather than always running it

Reply:
I get what you mean. I will add a flag to the settings object to prevent it from running all the time.